sdk/core/core-rest-pipeline/test/node/nodeHttpClient.spec.ts

```diff
@@ -71,7 +71,7 @@ describe("NodeHttpClient", function () {
     const client = createDefaultHttpClient();
     const clientRequest = createRequest();
     stubbedHttpsRequest.returns(clientRequest);
-    const request = createPipelineRequest({ url: "https://example.com" });
+    const request = createPipelineRequest({ url: "https://whateverhahaha.com" });
```

Comment:
lol is `example.com` banned or something?

Reply:
haha, will revert. I was debugging some issue and the request went to the wire and gave me 200, so I changed temporarily but forgot to restore.